test/scheduler_visualisation_test.py

```diff
@@ -292,7 +292,7 @@ def test_dep_graph_missing_deps(self):
 
         fail = dep_graph[BadReqTask(succeed=False).task_id]
         self.assertEqual(fail[u'name'], 'BadReqTask')
-        self.assertEqual(fail[u'params'], {'task_id': BadReqTask(succeed=False).task_id})
+        self.assertEqual(fail[u'params'], {'succeed': 'False'})
```

Comment:
I think we should display as much as we can, even if then the two kinds of UNKNOWN will give an inconsistent amount of information when you mouse-hover over them.